src/garage/torch/algos/__init__.py

```diff
@@ -1,4 +1,7 @@
 """PyTorch algorithms."""
+from garage.torch.algos._utils import (  # noqa: F401
```

Comment:
might as well just put them on their own lines each.

it seems to me that compute_advantages, filter_valids, and pad_to_last can call me public...

Reply:
actually @krzentner should be expose `make_optimizer` as well?

Reply:
All these function are only used by `garage.torch.algo` module, and were written for a specific use case. I think we should keep them private until we have a use for them elsewhere.